test/e2e/specs/editor/plugins/custom-post-types.spec.js

```diff
@@ -59,6 +59,42 @@ test.describe( 'Test Custom Post Types', () => {
 		await expect( parentPageLocator ).toHaveValue( parentPage );
 	} );
 
+	test( 'should not be able to rename a post that is either auto-drafted or lacks title support', async ( {
```

Comment:
Adding this e2e test was the first thing that occurred to me but now I wonder if it’d be better to create unit tests for [post-actions](https://github.com/WordPress/gutenberg/tree/trunk/packages/editor/src/components/post-actions) which currently lacks any.